### PR TITLE
 completed express-mutations

### PR DESCRIPTION
**Pull request by Bxvant**

### CODE REVIEW

**Review by RobertGardner — CHANGES_REQUESTED** [review]
One small bug.

File: express-mutations/server.ts [hunk 1]
```diff
@@ -83,8 +83,9 @@ app.delete('/api/actors/:actorId', async (req, res, next) => {
     const result = await db.query(sql, params);
     const actor = result.rows[0];
     if (!actor) {
-      throw new ClientError(404, ` actor ${actorId} does not exist`);
+      throw new ClientError(204, ` actor ${actorId} does not exist`);
```
Comment (RobertGardner) [inline express-mutations/server.ts:86]: Ooops, this one should be a 404. You send a 404 if the actor doesn't exist. You send a 204 if you successfully deleted the actor.

File: express-mutations/server.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    from "actors"
+    where "actorId" = $1
+    returning *;
+    `;
+    const params = [actorId];
+    const result = await db.query(sql, params);
+    const actor = result.rows[0];
+    if (!actor) {
+      throw new ClientError(404, ` actor ${actorId} does not exist`);
+    }
```
[the remark this answers is not in the document]
Comment (RobertGardner) [reply express-mutations/server.ts:87]: This is correct now.

**Review by RobertGardner — CHANGES_REQUESTED** [review]
Oops, a little too aggressive on the fix! ;-)

File: express-mutations/server.ts (hunk 1 above)
Comment (Bxvant) [reply express-mutations/server.ts]: i changed that on accident, forgot to change it back, whoops

**Review by RobertGardner — APPROVED** [review]

**Pull request merged**